shared_model/bindings/CMakeLists.txt

```diff
@@ -23,6 +23,7 @@ target_link_libraries(bindings
         shared_model_proto_backend
         shared_model_ed25519_sha3
         generator
+        shared_model_stateless_validation
```

Comment:
I think "stateless" shouldn't be here, otherwise it quite mixing up with torii

Reply:
What do you mean? Without this dependency build fails

Reply:
I meant only naming

Reply:
This name was already given to the target.